src/get-started/updating-your-code/index.md

```diff
@@ -174,10 +199,15 @@ The main change is changing variables from [snake_case](https://en.wikipedia.org
           You can now entirely replace the footer component by rendering your own,
           which can include a custom <code>navigation</code>
         </p>
-        <pre><code>{% raw %}
+
+```javascript
+{% raw %}
 {% block header %}
-    {{ govukFooter({ navigation: [] }) }}
-{% endblock %}{% endraw %}</code></pre>
+  {{ govukFooter({ navigation: [] }) }}
```

Comment:
Looks like there's an existing typo here – should be `{% block footer %}` on the line above!

Happy to fix in this PR whilst we're here or shall I raise a separate one?

Reply:
Ahh brilliant, thank you

Yep let's go and fix all the things at the same time

Reply:
It's not reflected in this comment, but fixed in commit https://github.com/alphagov/govuk-design-system/pull/2578/commits/056cd10adb65be12f324314399704a6d2cbbce74